tests/helpers/testhelpers.py

```diff
@@ -45,7 +45,7 @@ def generate_valid_test_cases():
 def generate_invalid_test_cases():
     invalid_files = list(INVALID_CASES_BASEDIR.glob("*.in"))
     ids = [str(f.relative_to(INVALID_CASES_BASEDIR)) for f in invalid_files]
-    return [f.read_text(encoding="utf8") for f in invalid_files], ids
+    return [f.absolute() for f in invalid_files], ids
```

Comment:
Why is this changed to `f.absolute()`? Why not just return the text?

Reply:
Before i removed the sanitize check from sanitize file, I think I used that, and then after removing the check i just fixed it, without realizing it was best to revert to how it was before, i will revert it.